migration/wrapper.go

```diff
@@ -0,0 +1,243 @@
+package migration
+
+// Package the default resources into binary data that is embedded in centrifuge
+// executable
+//
+//go:generate go-bindata -pkg migration -prefix "../" -o ./data.go ./files/...
```

Comment:
I think this will bloat our binary as the migration files increase. Maybe we can just inject the hashes to the data.go instead

Reply:
I am not certain about generating the data.go at all yet. I will keep you posted here.